src/client.cpp

```diff
@@ -334,8 +334,16 @@ void Client::resize_tiling(Rectangle rect, bool isFocused, bool minimalDecoratio
     dec->resize_outline(rect, scheme);
 }
 
-// from dwm.c
-bool Client::applysizehints(int *w, int *h) {
+/**
+ * @brief Update the given window size according to the client's size hints
+ * @param width
```

Comment:
w

Reply:
Oh, we don't actually state the name of the parameter here again, so "width" is the description of the parameter 'w'. Resolving this.